README.md

```diff
@@ -66,6 +66,19 @@ struct MyMetric {
 }
 ```
 
+Timestamps are optional and can be set via the `timestamp` attribute, if not set the current time will be used:
+
```

Comment:
can we also link to the timestamp documentation on influx? Its not obvious what the u64 is supposed to be within only these docs. Would be nice to add a short explanation here along with the link. No need to repeat everything they say but maybe mentioning its nanosecond-precision Unix time would be good here.

https://docs.influxdata.com/influxdb/v1.8/write_protocols/line_protocol_tutorial/#timestamp

Reply:
this should also be repeated in any code docstrings where timestamp is used